src/components/PatchVotes.jsx

```diff
@@ -0,0 +1,41 @@
+import { patchVotesById } from "../utils/api";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { GoThumbsup, GoThumbsdown } from "react-icons/go";
+
+const PatchVotes = ({ previsouVotes, review_id }) => {
+    const [votes, setVotes] = useState(previsouVotes)
+    const [err, setErr] = useState(null);
+    const navigate = useNavigate();
+
+    const thumbUp = () => {
+        setVotes((currentVotes) => currentVotes + 1);
+        setErr(null);
+        patchVotesById(1, review_id).catch((err) => {
+            setVotes((currentVotes) => currentVotes - 1);
```

Comment:
Really well handled! If you fancy a challenge see if you can limit it to one vote in either direction per page load :)